velox/common/caching/ScanTracker.h

```diff
@@ -78,9 +79,13 @@ struct TrackingData {
   int64_t readBytes{};
   int32_t numReferences{};
   int32_t numReads{};
-  void incrementReference(uint64_t bytes) {
+  void incrementReference(uint64_t bytes, int32_t quantum = 0) {
```

Comment:
It would be nice to document this method and explain the use of quantum parameter. I don't understand the new logic. Why would number of references depend on the load quantum?